crates/runestick/src/function.rs

```diff
@@ -23,6 +23,7 @@ where
     Tuple: From<Box<[V]>>,
 {
     inner: Inner<V>,
+    hash: Hash,
```

Comment:
IIUC, Rtti for the other variants have access to the corresponding type hashes, so this probably only needs to be added to the `FnHandler` and `FnOffset` variants.

Reply:
Fixed, thanks!